src/apiCalls.ts

```diff
@@ -18,13 +18,20 @@ export const getPlaylist = (genreArray: string[]) => {
       })
     )
     .then(data => {
-      const combinedPlaylist: CleanedAlbumTrack[] = data.flatMap(playlist => {
-        return playlist.tracks.track.map((song: AlbumTrack) => {
-          return cleanGenreTrackData(song);
-        });
-      });
+      const combinedPlaylist: Playlist = {
+        id: Date.now(),
+        name: genreArray.join(" "),
+        isSaved: false,
+        tracks: data.flatMap(playlist => {
+          return playlist.tracks.track.map((song: AlbumTrack) => {
+            return cleanGenreTrackData(song);
+          });
+        })
+      }
```

Comment:
Definitely think this was the correct call for being able to access the playlist in a new route. Initial thought is when a user creates a playlist that contains the same name (EX: a user clicks on the same genre twice either by accident or intentionally) how should this be handled?